### PR TITLE
fix: prevents the location from being lost

### PR DESCRIPTION
**Pull request by michaelphagen**
prevents the location from being set to the "none" option

prevents the location from being set to other where other is a
blank string

fixes a bug that caused the "Other" option in the location drop down
from holding on to the last set custom location, even in other forms,
until the browser was refreshed

fixes issue #13

### CODE REVIEW

File: html/Form.html [hunk 1]
```diff
@@ -73,8 +73,8 @@
               <option value="Edit Suite 2 509">Edit Suite 2 509</option>
               <option value="Lab 502">Lab 502</option>
               <option value="Lobby/Hallway">Lobby/Hallway</option>
-              <option value="Other">Other</option>
-              <option value="Studio 21 North" class="hidden"></option>
+              <option value="OtherSelector">Other</option>
```
Comment (dwmorrin) [inline html/Form.html]: Could you clarify this one?  It's not obvious why this change is needed to me, and the semantics are bothering me... the value of "OtherSelector" for an `<option>` within a `<select>` is not sitting right with me.

Comment (michaelphagen) [reply html/Form.html]: Is the explaination in issue #13 not clear? I agree the language is not the best. Perhaps otherDisplay or temp could replace the blank string one and "other" could remain "other"

Comment (dwmorrin) [reply html/Form.html]: is there no way to fix initializing the form in the javascript without changing the HTML?

Comment (dwmorrin) [reply html/Form.html]: I did not see what the root issue was.  I see that there is a bug that causes the location drop down to not get set properly... can we just add something to the Form's show() function to initialize things?

**Comment by michaelphagen** [comment]
There probably is, this was just the solution I came up with.nI can take 
another look tomorrow, I've been tearing down 21N and will probably do so 
until the end of my shift.
________________________________

Michael Hagen
Recorded Music Technician

Clive Davis Institute of Recorded Music
Tisch School of the Arts
New York University
194 Mercer St, #517
New York, NY 10012
914-299-3142

**Comment by michaelphagen** [comment]
The root issue was that once you set the other location in one form, the 
custom value would persist to other forms. I also encountered a bug where 
clicking the custom location wouldnt bring up the model window.
________________________________

Michael Hagen
Recorded Music Technician

Clive Davis Institute of Recorded Music
Tisch School of the Arts
New York University
194 Mercer St, #517
New York, NY 10012
914-299-3142

**Comment by dwmorrin** [comment]
Got it.  That can be corrected in js/Display.html

**Comment by dwmorrin** [comment]
My alternative was different enough I put it in a separate PR for consideration.

I added a helper function `resetLocation` to `app.pages.form`.  `displayForm` starts off by calling `resetLocation`. 

Additionally, when someone wants to edit the location, I noticed custom locations ended up with a blank select (not pretty), so I added a check that reverts the location select to the original "please select".  The hidden stored value for location remains whatever the custom value if the user just loses focus (doesn't select).

**Comment by michaelphagen** [comment]
Closing this PR, as we will be proceeding with PR #80
